hardware_interface/test/test_component_interfaces.cpp

```diff
@@ -13,730 +13,315 @@
+  hardware_interface::return_type configure(
+    const hardware_interface::HardwareInfo & /* info */) override
   {
-    info_ = sensor_info;
-    binary_to_voltage_factor_ = stod(info_.hardware_parameters["binary_to_voltage_factor"]);
-    status_ = status::CONFIGURED;
-    return return_type::OK;
+    // We hardcode the sensor info
+    return hardware_interface::return_type::OK;
   }
 
-  return_type start() override
+  std::vector<hardware_interface::StateHandle> export_state_handles() override
   {
-    if (status_ == status::CONFIGURED ||
-      status_ == status::STOPPED)
-    {
-      status_ = status::STARTED;
-    } else {
-      return return_type::ERROR;
-    }
-    return return_type::OK;
+    // We can read some voltage level
+    std::vector<hardware_interface::StateHandle> state_handles;
+    state_handles.emplace_back(
+      hardware_interface::StateHandle("joint1", "voltage", &voltage_level_));
+
+    return state_handles;
   }
 
-  return_type stop() override
+  hardware_interface::return_type start() override
   {
-    if (status_ == status::STARTED) {
-      status_ = status::STOPPED;
-    } else {
-      return return_type::ERROR;
-    }
-    return return_type::OK;
+    return hardware_interface::return_type::OK;
   }
 
-  status get_status() const override
+  hardware_interface::return_type stop() override
   {
-    return status_;
+    return hardware_interface::return_type::OK;
   }
 
-  return_type read_sensors(const std::vector<std::shared_ptr<components::Sensor>> & sensors) const
-  override
+  hardware_interface::status get_status() const override
   {
-    return_type ret = return_type::OK;
-    for (const auto & sensor : sensors) {
-      ret = sensor->set_state(ft_hw_values_);
-      if (ret != return_type::OK) {
-        break;
-      }
-    }
-    return ret;
+    return hardware_interface::status::UNKNOWN;
+  }
+
+  hardware_interface::return_type read() override
+  {
+    // no-op, static value
+    return hardware_interface::return_type::OK;
   }
 
 private:
-  HardwareInfo info_;
-  status status_ = status::UNKNOWN;
-  double binary_to_voltage_factor_;
-  std::vector<double> ft_hw_values_ = {1, -1.0, 3.4, 7.9, 5.5, 4.4};
+  double voltage_level_ = 0x666;
 };
 
-class DummySystemHardware : public SystemHardwareInterface
+class DummySystem : public hardware_interface::components::SystemInterface
 {
-  return_type configure(const HardwareInfo & system_info) override
+  hardware_interface::return_type configure(
+    const hardware_interface::HardwareInfo & /* info */) override
   {
-    info_ = system_info;
-    api_version_ = stod(info_.hardware_parameters["example_api_version"]);
-    hw_read_time_ = stod(info_.hardware_parameters["example_param_read_for_sec"]);
-    hw_write_time_ = stod(info_.hardware_parameters["example_param_write_for_sec"]);
-    status_ = status::CONFIGURED;
-    return return_type::OK;
+    // We hardcode the actuator info
```

Comment:
not really "actuator info", I suggest updating these comments to just use "info"

Reply:
it's been like that in the original PR already.

Reply:
done in 5a27bf8